webapp/client/src/pages/RegistrationPage.js

```diff
@@ -33,7 +33,16 @@ export default function RegistrationPage({
   return (
     <>
       <StepHeaderButtons />
-      <FormHeader {...stepHeader} />
+      <FormHeader
+        {...stepHeader}
+        intro={
```

Comment:
I think with this you're passing two times the intro to the FormHeader. First `stepHeader.intro` and then what you're giving in `intro={`. As we don't actually don't need the first one, we can just omit that and only pass `title={stepHeader.title}` and `intro={TRANS_COMPONENT}`

Reply:
I removed the `stepHeader.intro` in UnlockCodeRequestInputStep and make the intro property optional in `StepHeaderProps`. Within the FormHeader there is already a logic whether Intro is set or not.

Reply:
Yes, that makes total sense, thank you. 👍